fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -940,10 +940,7 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 		// test change coordinators and cluster description
 		// we randomly pick one process(not coordinator) and add it, in this case, it should always succeed
 		{
-			// choose a new description if configuration allows transactions across differently named clusters
-			state std::string new_cluster_description = SERVER_KNOBS->ENABLE_CROSS_CLUSTER_SUPPORT
-			                                                ? deterministicRandom()->randomAlphaNumeric(8)
-			                                                : cs.clusterKeyName().toString();
```

Comment:
Someone added the code to use the `cs` which is not in this scope but it's a `state` variable thus compiled.
This pr fixes this.